src/Page/Programmer/index.js

```diff
@@ -264,7 +264,7 @@ function Programmer() {
                 className="card p-3 d-flex flex-column mh-10 bg-bg-card-1 justify-content-top align-items-center text-left text-sm-center w-100 rounded border-0 "
                 onClick={(e) => {
                   e.preventDefault();
-                  window.open("https://admissions.navgurukul.org/", "_blank");
+                  window.open("https://www.merakilearn.org/admission", "_blank");
                 }}
```

Comment:
The code for this click handler is duplicated multiple times.  It's short, but if it's expected to say the same, perhaps a named function could be used instead to avoid this repetition.